js/sign/README.md

```diff
@@ -120,6 +124,17 @@ wbn-sign \
 -k ~/path/to/ed25519key.pem
 ```
 
+```bash
+wbn-sign \
+-i ~/path/to/webbundle.wbn \
+-o ~/path/to/signed-webbundle.swbn \
+-k ~/path/to/ed25519key.pem \
+-k ~/path/to/ecdsa_p256key.pem
+--version v2 \
+--web-bundle-id \
+  amfcf7c4bmpbjbmq4h4yptcobves56hfdyr7tm3doxqvfmsk5ss6maacai
+```
+
 ### Running wbn-dump-id
 
 There are the following command-line flags available:
```

Comment:
Same below: `--private-key` and `--with-iwa-scheme`.

Reply:
Oh, and the example for that is not correct too (see line 136 invoking `wbn-sign` instead of `wbn-dump-id`).

Reply:
Fixed